docker/compose/multi-dbnode/m3dbnode01.yml

```diff
@@ -0,0 +1,204 @@
+    throttle: 2m
+    checkInterval: 1m
+
+  pooling:
+    blockAllocSize: 16
+    type: simple
+    seriesPool:
+        size: 262144
+        lowWatermark: 0.7
+        highWatermark: 1.0
+    blockPool:
+        size: 262144
+        lowWatermark: 0.7
+        highWatermark: 1.0
+    encoderPool:
+        size: 262144
+        lowWatermark: 0.7
+        highWatermark: 1.0
+    closersPool:
+        size: 104857
+        lowWatermark: 0.7
+        highWatermark: 1.0
+    contextPool:
+        size: 262144
+        lowWatermark: 0.7
+        highWatermark: 1.0
+    segmentReaderPool:
+        size: 16384
+        lowWatermark: 0.7
+        highWatermark: 1.0
+    iteratorPool:
+        size: 2048
+        lowWatermark: 0.7
+        highWatermark: 1.0
+    fetchBlockMetadataResultsPool:
+        size: 65536
+        capacity: 32
+        lowWatermark: 0.7
+        highWatermark: 1.0
+    fetchBlocksMetadataResultsPool:
+        size: 32
+        capacity: 4096
+        lowWatermark: 0.7
+        highWatermark: 1.0
+    hostBlockMetadataSlicePool:
+        size: 131072
+        capacity: 3
+        lowWatermark: 0.7
+        highWatermark: 1.0
+    blockMetadataPool:
+        size: 65536
+        lowWatermark: 0.7
+        highWatermark: 1.0
+    blockMetadataSlicePool:
+        size: 65536
+        capacity: 32
+        lowWatermark: 0.7
+        highWatermark: 1.0
+    blocksMetadataPool:
+        size: 65536
+        lowWatermark: 0.7
+        highWatermark: 1.0
+    blocksMetadataSlicePool:
+        size: 32
+        capacity: 4096
+        lowWatermark: 0.7
+        highWatermark: 1.0
+    identifierPool:
+        size: 262144
+        lowWatermark: 0.7
+        highWatermark: 1.0
+    bytesPool:
+        buckets:
+            - capacity: 16
+              size: 524288
+              lowWatermark: 0.7
+              highWatermark: 1.0
+            - capacity: 32
+              size: 262144
+              lowWatermark: 0.7
+              highWatermark: 1.0
+            - capacity: 64
+              size: 131072
+              lowWatermark: 0.7
+              highWatermark: 1.0
+            - capacity: 128
+              size: 65536
+              lowWatermark: 0.7
+              highWatermark: 1.0
+            - capacity: 256
+              size: 65536
+              lowWatermark: 0.7
+              highWatermark: 1.0
+            - capacity: 1440
+              size: 16384
+              lowWatermark: 0.7
+              highWatermark: 1.0
+            - capacity: 4096
+              size: 8192
+              lowWatermark: 0.7
+              highWatermark: 1.0
+
+  config:
+    static:
+        topology:
+            shards: 32
+            replicas: 3
+            hosts:
+                - hostID: m3dbnode01
+                  listenAddress: "127.0.0.1:19001"
+                - hostID: m3dbnode02
+                  listenAddress: "127.0.0.1:29002"
+                - hostID: m3dbnode03
+                  listenAddress: "127.0.0.1:39003"
```

Comment:
Yeah we really should avoid using the static topology in examples.  Can we get @justinjc to help out crafting the embedded KV config for this multi-dbnode example?